src/chainparams.cpp

```diff
@@ -171,7 +171,6 @@ class CTestNetParams : public CMainParams {
         vAlertPubKey = ParseHex("044e7a1553392325c871c5ace5d6ad73501c66f4c185d6b0453cf45dec5a1322e705c672ac1a27ef7cdaf588c10effdf50ed5f95f85f2f54a5f6159fca394ed0c6");
         nDefaultPort = 18233;
         nMinerThreads = 0;
-        nMaxTipAge = 0x7fffffff;
```

Comment:
Are we going to add this back in?  What about setting it to 1 week?  It's useful in case testnet is unavailable (as has happened with Bitcoin) where people switch to mainnet leaving few or even zero nodes running on testnet.    If our testnet node continues to mine, it might not be a problem as it can create a minimum difficulty block.